disnake/abc.py

```diff
@@ -1290,9 +1290,16 @@ async def create_invite(
         max_age: :class:`int`
             How long the invite should last in seconds. If it's 0 then the invite
             doesn't expire. Defaults to ``0``.
+
+            .. warning::
+
+                If the guild is not a Community guild (has ``COMMUNITY`` in :attr:`.Guild.features`),
+                this must be set to a time between ``0`` and ``2_592_000`` seconds.
```

Comment:
imo it looks better without the underscores

Reply:
switched to commas in https://github.com/DisnakeDev/disnake/pull/1056/commits/9fef8f2cb74c76c4c1bff54a0558c750bacc364f

Reply:
removed commas entirely as per discord discussion 
[`8453d33` (#1056)](https://github.com/DisnakeDev/disnake/pull/1056/commits/8453d33e71808739ac50db537f0332f177d12829)